cpp/serve/grammar/grammar_state_matcher.cc

```diff
@@ -469,9 +469,10 @@ TVM_REGISTER_GLOBAL("mlc.serve.GrammarStateMatcherFromTokenizer")
 TVM_REGISTER_GLOBAL("mlc.serve.GrammarStateMatcherFromTokenTable")
     .set_body([](TVMArgs args, TVMRetValue* rv) {
       BNFGrammar grammar = args[0];
```

Comment:
cc @Ubospica Let me know if this change is fine for you; thank you!

Reply:
Yeah that looks cool. Thanks for the update